### PR TITLE
FIX: create_hfss_setup

### PR DESCRIPTION
**Pull request by gkorompi**

### CODE REVIEW

File: src/pyedb/dotnet/edb_core/utilities/simulation_setup.py [hunk 1]
```diff
@@ -56,9 +56,12 @@ def __init__(self, pedb, edb_setup=None):
             "kDDRwizard": None,
             "kQ3D": None,
             "kNumSetupTypes": None,
-            "kRaptorX": self._pedb.simsetupdata.RaptorX.RaptorXSimulationSettings,
             "kHFSSPI": self._pedb.simsetupdata.HFSSPISimulationSettings,
         }
+        if self._pedb.edbversion:
+            version = self._pedb.edbversion.split(".")
+            if int(version[0]) == 2024 and int(version[1]) == 2 or int(version[0]) > 2024:
+                self._setup_type_mapping.update({"kRaptorX": self._pedb.simsetupdata.RaptorX.RaptorXSimulationSettings})
```
Comment (SMoraisAnsys) [inline src/pyedb/dotnet/edb_core/utilities/simulation_setup.py:64]:
```suggestion
        }
        version = self._pedb.edbversion.split(".")
        if int(version[0]) == 2024 and int(version[1]) == 2 or int(version[0]) > 2024:
            self._setup_type_mapping.update({
                "kRaptorX": self._pedb.simsetupdata.RaptorX.RaptorXSimulationSettings,
                "kHFSSPI": self._pedb.simsetupdata.HFSSPISimulationSettings,
            })
```
As far as I know, kHFSSPI is also not allowed in 2023.2
Also, I removed the check on edbversion. Is there a reason for your if statement ?

**Review by SMoraisAnsys — CHANGES_REQUESTED** [review]

**Review by SMoraisAnsys — APPROVED** [review]
LGTM

**Review by svandenb-dev — DISMISSED** [review]
LGTM

**Review by SMoraisAnsys — APPROVED** [review]

**Review by svandenb-dev — APPROVED** [review]
LGTM

File: src/pyedb/dotnet/edb_core/utilities/simulation_setup.py [hunk 2]
```diff
@@ -103,10 +110,10 @@ def _set_edb_setup_info(self, edb_setup_info):
...
             "kNumSetupTypes": None,
         }
-        if self._pedb.edbversion:
-            version = self._pedb.edbversion.split(".")
-            if int(version[0]) == 2024 and int(version[1]) == 2 or int(version[0]) > 2024:
-                setup_type_mapping["kRaptorX"] = utility.RaptorXSimulationSetup
+
+        version = self._pedb.edbversion.split(".")
+        if int(version[0]) == 2024 and int(version[1]) == 2 or int(version[0]) > 2024:
+            setup_type_mapping["kRaptorX"] = utility.RaptorXSimulationSetup
```
Comment (SMoraisAnsys) [inline src/pyedb/dotnet/edb_core/utilities/simulation_setup.py:116]: Aren't we missing the mapping to "kHFSSPI" here ?

Comment (ring630) [reply src/pyedb/dotnet/edb_core/utilities/simulation_setup.py:116]: @gkorompi can you add kHFSSPI in another PR?

**Pull request merged**